src/mito2/src/sst/file_purger.rs

```diff
@@ -35,11 +38,98 @@ pub trait FilePurger: Send + Sync {
 
 pub type FilePurgerRef = Arc<dyn FilePurger>;
 
-// TODO(yingwen): Remove this once we implement the real purger.
-/// A purger that does nothing.
-#[derive(Debug)]
-struct NoopPurger {}
+pub struct LocalFilePurger {
+    pub scheduler: Arc<LocalScheduler>,
+
+    pub sst_layer: Arc<AccessLayer>,
```

Comment:
This two fields do not have to be public?

Reply:
may be not, I delete "pub" in two fields